tests/testthat/test-rstudio_markers.R

```diff
@@ -4,47 +4,56 @@ test_that("it returns markers which match lints", {
 
   lint1 <- structure(
     list(
-      Lint(filename = "test_file",
-           line_number = 1L,
-           column_number = 2L,
-           type = "error",
-           line = "a line",
-           message = "hi")
+      Lint(
+        filename = "test_file",
+        line_number = 1L,
+        column_number = 2L,
+        type = "error",
+        line = "a line",
+        message = "hi"
+      )
     ),
     class = "lints"
   )
+  lint1[[1L]]$linter <- "linter_name"
 
   marker1 <- rstudio_source_markers(lint1)
   expect_equal(marker1$name, "lintr")
   expect_equal(marker1$markers[[1L]]$type, lint1[[1L]]$type)
   expect_equal(marker1$markers[[1L]]$file, lint1[[1L]]$filename)
   expect_equal(marker1$markers[[1L]]$line, lint1[[1L]]$line_number)
   expect_equal(marker1$markers[[1L]]$column, lint1[[1L]]$column_number)
-  expect_equal(marker1$markers[[1L]]$message, lint1[[1L]]$message)
+  expect_equal(marker1$markers[[1L]]$message, paste0("[", lint1[[1L]]$linter, "] ", lint1[[1L]]$message))
```

Comment:
we really need some better test helpers... really hard to tell what the impact of the change is exactly.

something like `expect_lint_output(x, "...", rstudio = TRUE)`, where we do some `trim_some()`-like logic on `"..."` to maximize readability would make it super clear to the test reader and super easy for the test writer.

(for another day...)